assets/blocks/learner-courses-block/course-progress.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import { sprintf, _n } from '@wordpress/i18n';
+
+/**
+ * Course progress component.
+ *
+ * @param {Object} props
+ * @param {number} props.lessons         Number of lessons.
+ * @param {number} props.completed       Number of completed lessons.
+ * @param {number} props.hideProgressBar Whether hide progress bar.
+ */
+const CourseProgress = ( { lessons, completed, hideProgressBar } ) => {
```

Comment:
We should reuse the course progress block (its `CourseProgressEdit` component) here, since we need 90% the same markup & functionality. It seems fairly simple to refactor that to fit, but will be worth it's own PR.

Reply:
I remembered you commented that, and I checked it, but I didn't want to do it for now because the layout is different now. And depending on our decisions about the styling, we can have different approaches for styling each one, what would cause a component with many conditions only to reuse a few things.

Reply:
Let's use the same layout here too. We should not have two progress bar block components doing the exact same thing written twice from scratch. 
When we change the styling system at some point, we can update that shared component.

Reply:
I refactored that in a different PR to avoid conflicts: https://github.com/Automattic/sensei/pull/4149 ;)